routes.js

```diff
@@ -2,8 +2,26 @@ const express = require('express')
 const Router = express.Router()
 
 const todoController = require('./controllers/todo')
+const userController = require('./controllers/users')
+const commentsController = require('./controllers/comments')
 
-Router.get('/', todoController.getAll)
-Router.post('/', todoController.save)
+Router.post('/comments', commentsController.post)
+Router.get('/comments', commentsController.getAll)
+Router.put('/comments', commentsController.modify)
+Router.delete('/comments', commentsController.deleteById)
+
+Router.link('/user-comments', commentsController.getCommentsByUser)
+Router.link('/comments-todo', commentsController.getTodoByComment)
```

Comment:
user-comments and comments-todo does not match with restful specifications, you can use something like /user/setings and /todo/comments